### PR TITLE
fix #63281 cimri.com

### LINKED ISSUE #63281: cimri.com
### Issue URL (Ads)
 [https://www.cimri.com/klavye-mouse/en-ucuz-logitech-m330-silent-910-004909-siyah-kablosuz-mouse-fiyatlari,36549661](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.cimri.com%2Fklavye-mouse%2Fen-ucuz-logitech-m330-silent-910-004909-siyah-kablosuz-mouse-fiyatlari%2C36549661)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.12
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (1440), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by stanislav-atr**
#63281

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -58,6 +58,7 @@ gazetebirlik.com###masthead-reklam
 gazetebirlik.com##.reklam-button
 acikradyo.com.tr##.reklam-item
 teknodestek.com.tr##div[style="float: none; margin:10px 0 10px 0; text-align:center;"]
+cimri.com##div[class] > a[href^="https://www.ciceksepeti.com/?"]
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:61]: why not just `a[href^="https://www.ciceksepeti.com/?"]`?

**Review by CemTR — CHANGES_REQUESTED** [review]
Rule not working.

**Comment by zloyden** [comment]
@CemTR please check this `cimri.com###__next > div > div[class*="-"] > a[rel^="nofollow"][target="_blank"]`

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (stanislav-atr) [reply TurkishFilter/sections/specific.txt:61]: They have already changed url.

**Comment by stanislav-atr** [comment]
> cimri.com###__next > div > div[class*="-"] > a[rel^="nofollow"][target="_blank"]

@zloyden that works, should i add it?

**Comment by zloyden** [comment]
Yes, exchange your rule to this one. It works fine on my end.

**Review by zloyden — APPROVED** [review]

**Pull request merged**